<Type Name="FieldAccessException" FullName="System.FieldAccessException">
  <Metadata><Meta Name="ms.openlocfilehash" Value="14f03e387488171716f16b11492fc8a122b0b2f4" /><Meta Name="ms.sourcegitcommit" Value="11d168140aa8fade0768c2a9dde3e3bcacfdfb7d" /><Meta Name="ms.translationtype" Value="MT" /><Meta Name="ms.contentlocale" Value="ja-JP" /><Meta Name="ms.lasthandoff" Value="05/27/2020" /><Meta Name="ms.locfileid" Value="83967963" /></Metadata><TypeSignature Language="C#" Value="public class FieldAccessException : MemberAccessException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi beforefieldinit FieldAccessException extends System.MemberAccessException" FrameworkAlternate="dotnet-uwp-10.0;net-5.0;netcore-1.0;netcore-1.1;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netcore-3.1;netstandard-1.3;netstandard-1.4;netstandard-1.5;netstandard-1.6;netstandard-2.0;netstandard-2.1" />
  <TypeSignature Language="DocId" Value="T:System.FieldAccessException" />
  <TypeSignature Language="VB.NET" Value="Public Class FieldAccessException&#xA;Inherits MemberAccessException" />
  <TypeSignature Language="C++ CLI" Value="public ref class FieldAccessException : MemberAccessException" />
  <TypeSignature Language="F#" Value="type FieldAccessException = class&#xA;    inherit MemberAccessException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable beforefieldinit FieldAccessException extends System.MemberAccessException" FrameworkAlternate="netframework-1.1;netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8;xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0" />
  <AssemblyInfo>
    <AssemblyName>System.Runtime</AssemblyName>
    <AssemblyVersion>4.0.20.0</AssemblyVersion>
    <AssemblyVersion>4.1.0.0</AssemblyVersion>
    <AssemblyVersion>4.2.0.0</AssemblyVersion>
    <AssemblyVersion>4.2.1.0</AssemblyVersion>
    <AssemblyVersion>4.2.2.0</AssemblyVersion>
    <AssemblyVersion>5.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <AssemblyInfo>
    <AssemblyName>mscorlib</AssemblyName>
    <AssemblyVersion>1.0.5000.0</AssemblyVersion>
    <AssemblyVersion>2.0.0.0</AssemblyVersion>
    <AssemblyVersion>2.0.5.0</AssemblyVersion>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <AssemblyInfo>
    <AssemblyName>netstandard</AssemblyName>
    <AssemblyVersion>2.0.0.0</AssemblyVersion>
    <AssemblyVersion>2.1.0.0</AssemblyVersion>
  </AssemblyInfo>
  <TypeForwardingChain>
    <TypeForwarding From="mscorlib" FromVersion="4.0.0.0" To="System.Runtime" ToVersion="0.0.0.0" FrameworkAlternate="dotnet-uwp-10.0" />
    <TypeForwarding From="netstandard" FromVersion="2.1.0.0" To="System.Runtime" ToVersion="5.0.0.0" FrameworkAlternate="net-5.0" />
  </TypeForwardingChain>
  <Base>
    <BaseTypeName>System.MemberAccessException</BaseTypeName>
  </Base>
  <Interfaces />
  <Attributes>
    <Attribute FrameworkAlternate="netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0;netframework-4.8">
      <AttributeName>System.Runtime.InteropServices.ComVisible(true)</AttributeName>
    </Attribute>
    <Attribute FrameworkAlternate="netframework-1.1;netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8;xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0">
      <AttributeName>System.Serializable</AttributeName>
    </Attribute>
  </Attributes>
  <Docs>
    <summary><span data-ttu-id="229ec-101">クラス内の private フィールドや protected フィールドへの無効なアクセスを試みると、例外がスローされます。</span><span class="sxs-lookup"><span data-stu-id="229ec-101">The exception that is thrown when there is an invalid attempt to access a private or protected field inside a class.</span></span></summary>
    <remarks>
      <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="229ec-102">この例外は、通常、クラスライブラリ内のフィールドのアクセスレベルが変更され、そのライブラリを参照している1つ以上のアセンブリが再コンパイルされていない場合にスローされます。</span><span class="sxs-lookup"><span data-stu-id="229ec-102">This exception is typically thrown when the access level of a field in a class library is changed, and one or more assemblies referencing the library have not been recompiled.</span></span>  
  
 <span data-ttu-id="229ec-103"><xref:System.FieldAccessException> では、値 0x8013150 7 の HRESULT COR_E_FIELDACCESS を使用します。</span><span class="sxs-lookup"><span data-stu-id="229ec-103"><xref:System.FieldAccessException> uses the HRESULT COR_E_FIELDACCESS, which has the value 0x80131507.</span></span>  
  
 <span data-ttu-id="229ec-104">インスタンスの初期プロパティ値の一覧については<xref:System.FieldAccessException>を参照してください、<xref:System.FieldAccessException.%23ctor%2A>コンス トラクター。</span><span class="sxs-lookup"><span data-stu-id="229ec-104">For a list of initial property values for an instance of <xref:System.FieldAccessException>, see the <xref:System.FieldAccessException.%23ctor%2A> constructors.</span></span>  
  
> [!NOTE]
>  <span data-ttu-id="229ec-105">この例外は、 [Windows ストアアプリ用 .net](https://go.microsoft.com/fwlink/?LinkID=247912)または[ポータブルクラスライブラリ](/dotnet/standard/cross-platform/cross-platform-development-with-the-portable-class-library)には含まれていませんが、の一部のメンバーによってスローされます。</span><span class="sxs-lookup"><span data-stu-id="229ec-105">This exception is not included in the [.NET for Windows Store apps](https://go.microsoft.com/fwlink/?LinkID=247912) or the [Portable Class Library](/dotnet/standard/cross-platform/cross-platform-development-with-the-portable-class-library), but it is thrown by some members that are.</span></span> <span data-ttu-id="229ec-106">そのような場合に例外をキャッチするには、代わりに <xref:System.MemberAccessException> 用の `catch` ステートメントを記述します。</span><span class="sxs-lookup"><span data-stu-id="229ec-106">To catch the exception in that case, write a `catch` statement for <xref:System.MemberAccessException> instead.</span></span>  
  
 ]]></format>
    </remarks>
    <altmember cref="T:System.Exception" />
    <related type="Article" href="/dotnet/standard/exceptions/"><span data-ttu-id="229ec-107">例外の処理とスロー</span><span class="sxs-lookup"><span data-stu-id="229ec-107">Handling and Throwing Exceptions</span></span></related>
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.0.20.0</AssemblyVersion>
        <AssemblyVersion>4.1.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary><span data-ttu-id="229ec-108"><see cref="T:System.FieldAccessException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="229ec-108">Initializes a new instance of the <see cref="T:System.FieldAccessException" /> class.</span></span></summary>
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public FieldAccessException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.FieldAccessException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; FieldAccessException();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.0.20.0</AssemblyVersion>
        <AssemblyVersion>4.1.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
        <AssemblyVersion>4.2.2.0</AssemblyVersion>
        <AssemblyVersion>5.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>1.0.5000.0</AssemblyVersion>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.1.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters />
      <Docs>
        <summary><span data-ttu-id="229ec-109"><see cref="T:System.FieldAccessException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="229ec-109">Initializes a new instance of the <see cref="T:System.FieldAccessException" /> class.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="229ec-110">このコンストラクターは、新しいインスタンスの <xref:System.Exception.Message%2A> プロパティを、"呼び出し元がアクセスできないフィールドにアクセスしようとしました。" などのエラーを説明するシステム提供のメッセージに初期化します。</span><span class="sxs-lookup"><span data-stu-id="229ec-110">This constructor initializes the <xref:System.Exception.Message%2A> property of the new instance to a system-supplied message that describes the error, such as "Attempted to access a field that is not accessible by the caller."</span></span> <span data-ttu-id="229ec-111">このメッセージでは、現在のシステムのカルチャが考慮されます。</span><span class="sxs-lookup"><span data-stu-id="229ec-111">This message takes the current system culture into account.</span></span>  
  
 <span data-ttu-id="229ec-112">次の表に、<xref:System.FieldAccessException> のインスタンスに対するプロパティの初期値を示します。</span><span class="sxs-lookup"><span data-stu-id="229ec-112">The following table shows the initial property values for an instance of <xref:System.FieldAccessException>.</span></span>  
  
|<span data-ttu-id="229ec-113">プロパティ</span><span class="sxs-lookup"><span data-stu-id="229ec-113">Property</span></span>|<span data-ttu-id="229ec-114">値</span><span class="sxs-lookup"><span data-stu-id="229ec-114">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="229ec-115">null 参照 (Visual Basic の場合は `Nothing`)。</span><span class="sxs-lookup"><span data-stu-id="229ec-115">A null reference (`Nothing` in Visual Basic).</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="229ec-116">ローカライズされたエラー メッセージの文字列です。</span><span class="sxs-lookup"><span data-stu-id="229ec-116">The localized error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public FieldAccessException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.FieldAccessException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; FieldAccessException(System::String ^ message);" />
      <MemberSignature Language="F#" Value="new FieldAccessException : string -&gt; FieldAccessException" Usage="new System.FieldAccessException message" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.0.20.0</AssemblyVersion>
        <AssemblyVersion>4.1.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
        <AssemblyVersion>4.2.2.0</AssemblyVersion>
        <AssemblyVersion>5.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>1.0.5000.0</AssemblyVersion>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.1.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="229ec-117">例外の原因を示すエラー メッセージです。</span><span class="sxs-lookup"><span data-stu-id="229ec-117">The error message that explains the reason for the exception.</span></span></param>
        <summary><span data-ttu-id="229ec-118">指定したエラー メッセージを使用して、<see cref="T:System.FieldAccessException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="229ec-118">Initializes a new instance of the <see cref="T:System.FieldAccessException" /> class with a specified error message.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="229ec-119">このコンストラクターは、`message` パラメーターを使用して、新しいインスタンスの <xref:System.Exception.Message%2A> プロパティを初期化します。</span><span class="sxs-lookup"><span data-stu-id="229ec-119">This constructor initializes the <xref:System.Exception.Message%2A> property of the new instance by using the `message` parameter.</span></span> <span data-ttu-id="229ec-120">`message` の内容は、ユーザーが理解できる内容にします。</span><span class="sxs-lookup"><span data-stu-id="229ec-120">The content of `message` is intended to be understood by humans.</span></span> <span data-ttu-id="229ec-121">このコンストラクターの呼び出し元は、この文字列が現在のシステムのカルチャに合わせてローカライズ済みであることを確認しておく必要があります。</span><span class="sxs-lookup"><span data-stu-id="229ec-121">The caller of this constructor must ensure that this string has been localized for the current system culture.</span></span>  
  
 <span data-ttu-id="229ec-122">次の表に、<xref:System.FieldAccessException> のインスタンスに対するプロパティの初期値を示します。</span><span class="sxs-lookup"><span data-stu-id="229ec-122">The following table shows the initial property values for an instance of <xref:System.FieldAccessException>.</span></span>  
  
|<span data-ttu-id="229ec-123">プロパティ</span><span class="sxs-lookup"><span data-stu-id="229ec-123">Property</span></span>|<span data-ttu-id="229ec-124">値</span><span class="sxs-lookup"><span data-stu-id="229ec-124">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="229ec-125">null 参照 (Visual Basic の場合は `Nothing`)。</span><span class="sxs-lookup"><span data-stu-id="229ec-125">A null reference (`Nothing` in Visual Basic).</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="229ec-126">ローカライズされたエラー メッセージの文字列です。</span><span class="sxs-lookup"><span data-stu-id="229ec-126">The localized error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="protected FieldAccessException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);" />
      <MemberSignature Language="ILAsm" Value=".method familyhidebysig specialname rtspecialname instance void .ctor(class System.Runtime.Serialization.SerializationInfo info, valuetype System.Runtime.Serialization.StreamingContext context) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.FieldAccessException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
      <MemberSignature Language="VB.NET" Value="Protected Sub New (info As SerializationInfo, context As StreamingContext)" />
      <MemberSignature Language="C++ CLI" Value="protected:&#xA; FieldAccessException(System::Runtime::Serialization::SerializationInfo ^ info, System::Runtime::Serialization::StreamingContext context);" />
      <MemberSignature Language="F#" Value="new FieldAccessException : System.Runtime.Serialization.SerializationInfo * System.Runtime.Serialization.StreamingContext -&gt; FieldAccessException" Usage="new System.FieldAccessException (info, context)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>1.0.5000.0</AssemblyVersion>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.1.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
        <AssemblyVersion>4.2.2.0</AssemblyVersion>
        <AssemblyVersion>5.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Attributes>
        <Attribute FrameworkAlternate="netframework-4.0">
          <AttributeName>System.Security.SecuritySafeCritical</AttributeName>
        </Attribute>
      </Attributes>
      <Parameters>
        <Parameter Name="info" Type="System.Runtime.Serialization.SerializationInfo" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netframework-1.1;netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8;netstandard-2.0;xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0;netstandard-2.1;netcore-3.1;net-5.0" />
        <Parameter Name="context" Type="System.Runtime.Serialization.StreamingContext" Index="1" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netframework-1.1;netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8;netstandard-2.0;xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0;netstandard-2.1;netcore-3.1;net-5.0" />
      </Parameters>
      <Docs>
        <param name="info"><span data-ttu-id="229ec-127">シリアル化されたオブジェクトのデータを保持するオブジェクトです。</span><span class="sxs-lookup"><span data-stu-id="229ec-127">The object that holds the serialized object data.</span></span></param>
        <param name="context"><span data-ttu-id="229ec-128">送信元または送信先に関するコンテキスト情報。</span><span class="sxs-lookup"><span data-stu-id="229ec-128">The contextual information about the source or destination.</span></span></param>
        <summary><span data-ttu-id="229ec-129">シリアル化されたデータを使用して、<see cref="T:System.FieldAccessException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="229ec-129">Initializes a new instance of the <see cref="T:System.FieldAccessException" /> class with serialized data.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="229ec-130">このコンストラクターは、逆シリアル化中に、ストリームで送信された例外オブジェクトを再構築するために呼び出されます。</span><span class="sxs-lookup"><span data-stu-id="229ec-130">This constructor is called during deserialization to reconstitute the exception object that was transmitted over a stream.</span></span>  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public FieldAccessException (string message, Exception inner);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception inner) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.FieldAccessException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, inner As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; FieldAccessException(System::String ^ message, Exception ^ inner);" />
      <MemberSignature Language="F#" Value="new FieldAccessException : string * Exception -&gt; FieldAccessException" Usage="new System.FieldAccessException (message, inner)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.0.20.0</AssemblyVersion>
        <AssemblyVersion>4.1.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
        <AssemblyVersion>4.2.2.0</AssemblyVersion>
        <AssemblyVersion>5.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>1.0.5000.0</AssemblyVersion>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
        <AssemblyVersion>2.1.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="inner" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="229ec-131">例外の原因を示すエラー メッセージです。</span><span class="sxs-lookup"><span data-stu-id="229ec-131">The error message that explains the reason for the exception.</span></span></param>
        <param name="inner"><span data-ttu-id="229ec-132">現在の例外の原因になった例外です。</span><span class="sxs-lookup"><span data-stu-id="229ec-132">The exception that is the cause of the current exception.</span></span> <span data-ttu-id="229ec-133"><paramref name="inner" /> パラメーターが <see langword="null" /> でない場合は、内部例外を処理する <see langword="catch" /> ブロックで現在の例外が発生します。</span><span class="sxs-lookup"><span data-stu-id="229ec-133">If the <paramref name="inner" /> parameter is not <see langword="null" />, the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span></span></param>
        <summary><span data-ttu-id="229ec-134">指定したエラー メッセージおよびこの例外の原因となった内部例外への参照を使用して、<see cref="T:System.FieldAccessException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="229ec-134">Initializes a new instance of the <see cref="T:System.FieldAccessException" /> class with a specified error message and a reference to the inner exception that is the cause of this exception.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="229ec-135">前回の例外の直接的な結果としてスローされる例外は、<xref:System.Exception.InnerException%2A> プロパティに前回の例外への参照を含める必要があります。</span><span class="sxs-lookup"><span data-stu-id="229ec-135">An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property.</span></span> <span data-ttu-id="229ec-136"><xref:System.Exception.InnerException%2A> プロパティは、コンストラクターに渡されたものと同じ値を返します。または、コンストラクターに内部例外値が指定されていない場合は `null` を返します。</span><span class="sxs-lookup"><span data-stu-id="229ec-136">The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or returns `null` if it does not supply the inner exception value to the constructor.</span></span> <span data-ttu-id="229ec-137">このコンストラクターの呼び出し元は、この文字列が現在のシステムのカルチャに合わせてローカライズ済みであることを確認しておく必要があります。</span><span class="sxs-lookup"><span data-stu-id="229ec-137">The caller of this constructor must ensure that this string has been localized for the current system culture.</span></span>  
  
 <span data-ttu-id="229ec-138">次の表に、<xref:System.FieldAccessException> のインスタンスに対するプロパティの初期値を示します。</span><span class="sxs-lookup"><span data-stu-id="229ec-138">The following table shows the initial property values for an instance of <xref:System.FieldAccessException>.</span></span>  
  
|<span data-ttu-id="229ec-139">プロパティ</span><span class="sxs-lookup"><span data-stu-id="229ec-139">Property</span></span>|<span data-ttu-id="229ec-140">値</span><span class="sxs-lookup"><span data-stu-id="229ec-140">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="229ec-141">内部例外の参照です。</span><span class="sxs-lookup"><span data-stu-id="229ec-141">The inner exception reference.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="229ec-142">ローカライズされたエラー メッセージの文字列です。</span><span class="sxs-lookup"><span data-stu-id="229ec-142">The localized error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
  </Members>
</Type>
